src/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface ChromaticConfig {
```

Comment:
Actually, as per my previous comment, this is probably the place to document each of the individual options.

Reply:
@jwir3 added! 03eecbf9cd73d7891303b82a16f270706b2964ce